packages/sit-onyx/src/components/OnyxDataGrid/features/index.ts

```diff
@@ -201,6 +202,7 @@ export type DataGridFeature<
     }[];
     wrapper?: (column: PublicNormalizedColumnConfig<TEntry>) => Component;
   };
+  attributes?: () => HTMLAttributes;
```

Comment:
Please also update the naming here to keep it consistent:
```suggestion
  scrollContainerAttributes?: () => HTMLAttributes;
```